<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FourthScheme</class>
 <widget class="QMainWindow" name="FourthScheme">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>225</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>240</width>
    <height>225</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>240</width>
    <height>225</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>FourthScheme</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Значение:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QSpinBox" name="RowSpinBox">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>1000</number>
      </property>
      <property name="value">
       <number>3</number>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Строка:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QSpinBox" name="ValueSpinBox">
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>1000</number>
      </property>
      <property name="stepType">
       <enum>QAbstractSpinBox::DefaultStepType</enum>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QSpinBox" name="ColumnSpinBox">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>1000</number>
      </property>
      <property name="value">
       <number>3</number>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QPushButton" name="OpenButton">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>Открыть</string>
       </property>
      </widget>
      <widget class="QPushButton" name="SaveButton">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>Сохранить</string>
       </property>
      </widget>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Столбец:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="AddButton">
      <property name="text">
       <string>Добавить</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="DelButton">
      <property name="text">
       <string>Удалить</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>240</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
